kclvm/tools/src/LSP/src/lib.rs

```diff
@@ -13,6 +13,7 @@ mod hover;
 mod main_loop;
 mod notification;
 mod quick_fix;
+pub mod rename;
```

Comment:
Have we already referenced the implementation `rename_symbol` in kclvm/api/src/service/service_impl here? I couldn't find it in the main branch or in this PR.

Reply:
fixed, ptal

Reply:
> fixed, ptal

I mean that there is a impl todo at https://github.com/kcl-lang/kcl/blob/main/kclvm/api/src/service/service_impl.rs#L481